addons/mail/static/src/core/common/composer.js

```diff
@@ -379,42 +379,36 @@ export class Composer extends Component {
                 return {
                     ...props,
                     optionTemplate: "mail.Composer.suggestionThread",
-                    options: suggestions.map((suggestion) => {
-                        return {
-                            label: suggestion.parent_channel_id
-                                ? `${suggestion.parent_channel_id.displayName} > ${suggestion.displayName}`
-                                : suggestion.displayName,
-                            thread: suggestion,
-                            classList: "o-mail-Composer-suggestion",
-                        };
-                    }),
+                    options: suggestions.map((suggestion) => ({
+                        label: suggestion.parent_channel_id
+                            ? `${suggestion.parent_channel_id.displayName} > ${suggestion.displayName}`
```

Comment:
The duality between `display_name` in Python and `displayName` in the JavaScript code is awkward, especially since the basic case for `displayName` returns `name`.

Reply:
What name do you suggest for the JS field? I don't really have a better idea.

As I don't really have it in my diff (there a lot of occurrences), I prefer if we do another PR if we find a good name.